packages/turbo-ignore/utils.js

```diff
@@ -0,0 +1,69 @@
+const fs = require("fs");
+const path = require("path");
+const { findRootSync } = require("@manypkg/find-root");
+
+export function searchUp(pathName, cwd) {
```

Comment:
```suggestion
export function searchUp(searchTarget, cwd) {
```

Or something similar?